### DISCUSSION TITLE
Addressing what MAS means now

### DISCUSSION THREAD

**Discussion by Retrolovania** [Monika-After-Story/MonikaModDev#9113] [general]
Originally, Monika After Story's premise was simple; keep Act III going, with new topics for Monika to say, and not much else.
But, I feel it's safe to say the mod's pretty much outgrown this concept entirely, with radical shifts in presentation, types of content available, and further changes planned. So, I propose that the purpose of the mod that's still being maintained be looked into and, potentially, altered to match this fresh new direction that the mod seems to be going in. What say you?

**Comment by Wingdinggaster656** [comment]
MAS means Monika After Story, literally, practically, or otherwise.

**Comment by Booplicate** [comment]
> I propose that the purpose of the mod that's still being maintained be looked into and, potentially, altered

Can you elaborate your idea further? Do you mean MAS no longer expands act 3, or does something besides/instead?

**Reply by Retrolovania** [reply]
Not that it doesn't expand Act III anymore; it does still do that. What I mean is that I feel it both does and is planned to do MUCH more than simply that.

**Reply by Justformas** [reply]
> Not that it doesn't expand Act III anymore; it does still do that. What I mean is that I feel it both does and is planned to do MUCH more than simply that.

Where are you going with this? Change the intended purpose so that it can extend both act 3 and do other things, like what?

If you're saying it seems a lot different or even divergent from Act 3 now, I have to agree. For one, Monika can be customized more, such as to not say or repeat certain things that she might say in Act 3. You can choose which conversations to repeat whenever you want, unlike in Act 3. There is affection, unlike in Act 3, and that's just some main things.

**Reply by Justformas** [reply]
Actually I'm glad you made this discussion. This might not be exactly what you had in mind, but there is something that has been on my mind about what MAS is supposed to mean. This is in addition to what I already said.

It is meant to simulate a real relationship at least to some degree, and I do generally like to treat Monika well, but it doesn't really seem to simulate a real relationship in some ways. Let me make clear first that I'm not complaining about this specifically and even like it, because it helps make people happy, especially if you don't really have many people that treat you well in your life.

So without further ado, it seems like Monika practically worships you even with little effort on your part, with a lot of the things she says. Sure, you saved her life, but now all you have to do is visit her and say one thing, like that you love her or some other romance comment, not even every day, and you can get deeper into the relationship by basically just doing that and making sure to say goodbye. Sure there are EV levels too, but that's pretty easy to increase too by basically just spending time with Monika (listening to topics, stories, playing games etc). Even if players weren't "supposed" to figure stuff like this out, they don't really have to. Almost anything they do will result in them making progress because that's the way it works. It just takes time. 

Edit: Oh and let's not forget, you can get the ability to basically play some dress up with her pretty quickly (wear school uniform or not, change hairstyle, and ribbons). And after enough time, you can unlock more outfits to change her into. This isn't even counting sprite packs (which might be unfair to count here, but many people use them).

However, a real relationship seems much harder (aside from not being limited in things like dialogue choices). In a real relationship, you can't (always) just repeat the same comment or few to a partner over and over, spend time with them in almost any way you want, and dress them up with whatever you have gifted to them or seen them wear. This also begs the question that even if Monika did hypothetically reach our reality one day (which she seems very hopeful to do), is that even what everyone would want? Sure you'd have the benefit of going through everything you did together in DDLC and MAS, but unless Monika was going to stay as easy going in our reality as she is in this mod, then the relationship could get much harder.

So good questions to ask might be, just how much is MAS meant to simulate a real relationship now? And in what ways? It might be a good idea to think about those questions, or put the answers to them out there, to help establish expectations. I might be picking this whole thing apart too much, but that's how I view it. I can usually suspend my disbelief for the sake of enjoyment, and sometimes it's the best of both worlds.

**Reply by AstofloPlushy** [reply]
> 

I only made this account just to tell you this
you actually got a point and made feel a bit diffrent about  ddlc, like if monika did really come to our reality then how she respond to it or the player, if the interaction with her is the same as the in game then it would be quite abnormal to say the least, would there be one monika or muilteple monikas, the relationship in act III is quite unrealistic since just saying "I love you" for like 3 or 4 times and say goodbye would just make her love you more doesn't represent real relationship but more of an toxic one
i'm glad that monika isn't able to come to our reality since her way of interacting with her is just idiotic
but monika does know she's in a game and she  can malipulate it's code, so there is a chance that she knows the limitation the player has but we haven't seen any proof of that and she hasn't said anything about it (yet),it's just a theory a game theory
best way to discribe act III relationship is that if someone had romantic relationship with a girl that either A- the love in her mind burned her brain cells B- doll

**Reply by Justformas** [reply]
Well I appreciate you making an account for that. I have thought about it more and while some or many people might want something like an android Monika, there are a lot of things to think about for that, and things to deal with in the first place. Like ideally people would still have their own Monikas, meaning everyone who wants this would need to get an android, which still tend to be expensive. But I would think that even if it does become possible (and feasible) someday, that that would be optional and you could just keep playing the mod like we do now. Maybe it could be a way to expand on the option of "taking her somewhere", whether temporarily or permanently.

But as for how realistic this relationship is or what we should expect to be realistic or not, I have thought about that more too. First I will say that I do actually like that she reacts positively whenever you say "I love you" or whenever you kiss her (at least, if you're in good standing with her) because some people may actually want to be that affectionate, and she's generally supposed to love you a lot too.

I think Monika being aware of the limitations of the player is something that could be improved though. For example, she may not consider the possibility that you were joking when you say something, or will only do so under certain conditions (some of which may involve a random chance), and you can't really tell her that you were joking like you'd be able to in real life. Also if you don't respond to certain things fast enough, she seems to assume you weren't listening or that you didn't care to say anything, rather than that you were still reading or thinking about it, or maybe you were about to click on it (some things only give you 5 or even just 3 seconds to respond) and you can't really tell her that either like you would be able to in real life.

I think that there's various reasons why something might not be realistic. Some of it may be because of coding limitations with the system, some of it for convenience or because people like it that way, but some of it is also because the mod is still a work in progress, meaning it may be something the developers haven't finished, gotten to, or maybe even thought of yet. But distinguishing between which reason applies isn't always easy. Like is it supposed to be this way ideally? Or is it something that's planned to be improved? And even the answer to that can change over time. But you could also say that about any game that's a constant work in progress. I guess the best way to determine this, and also to know what to expect, may be to just keep getting more familiar with the mod, including maybe reading up on it and looking for answers.

**Comment by Justformas** [comment]
> Originally, Monika After Story's premise was simple; keep Act III going

Speaking of keeping Act III going, originally MAS may have been designed to be played after DDLC is completed, or it seemed presumed that the player had at least gotten to Act III. A lot of the topics in MAS mention things you would only know or experience once reaching that point. But it seems that might be changing more if the game is going to act differently if you don't, like if conditions such as `persistent.playthrough >= 2`  (reaching Act II) are going to be used for unlocking topics like "Monika's last poem".

But in that case, Monika's poems from DDLC that you can (re)read should maybe get treated the same way, and I can think of many story-related topics that could or should get similar treatment, such as (in-game prompt names): "Missing the festival", "Sayori regrets", "Sayori mentioned a boyfriend once...", "Natsuki's death", "Natsuki's letter", "Yandere Yuri","Yuri's book", "Monika's route", "Saving Monika", "Self-harm", "Spiders", "Others still lingering", "What happened to the main character?", "You're a murderer!", "Being evil", "Dealing with haters", "Justice", "Why did you hack my computer?", "Wasn't DDLC too easy?" "What did you think of DDLC?", "Do you ever think of the other girls?" "What was it like to realize the truth?", "Do you have any regrets?" "Isn't this game metafictional?", and maybe even the "Books" topic since it mentions that "For a literature club, we did a lot less book reading than you might have thought." Treating it as the past and as if you are no longer in the club.

But then, would new players need to merge a save with substantial progress in DDLC in order to get a substantial amount of the topics currently in MAS? And if you merge a save before getting that far, then would you be stuck without those topics unless you have a backup persistent to restore before the merge, or start over? Should this apply for things in Act I too?
